### ISSUE TITLE
A11y_ Azure Tools for VS Code_Create New Project_Select a Language_Voiceover:When the voiceover focus moves to "View sample projects' control inside 'Select a language', the control is being announced as simple text element instead of link.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/vscode-azurefunctions#3158]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_AzureToolsforVSCodeMAC_May2022;#Azure Tools for VS Code;#MAC;#FTP;#DesktopApp;#Voiceover;#WCAG4.1.2;#AINotInScope;#LinkedBug;#LinkedBug149305;#External:Apple;#closed;

## Environment Details:

Application Name: Azure Tools for VS Code
Application Version: 1.66.2 (Universal)
Mac Version: macOS Monterey 12.3.1
Screen Reader: Voiceover

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch Visual Studio Code.
3. Click on extension button and search for "Azure Functions".
4. Click on install and tab to the "Azure" button.
5. Tab till 'Functions' button and hit enter.
6. Tab till 'Create new project' button and hit enter.
7. Tab till 'View Sample Projects' under 'Select a language'.
8. Use Voiceover to observe.

## Actual:
When the voiceover focus moves to "View sample projects' control inside 'Select a language', the control is being announced as simple text element instead of link.

### Same issue can be observed for the following scenario:

1. Create New Project_Change Template Filter (The control should be announced as button)

## Expected:
When the voiceover focus moves to "View sample projects' control inside 'Select a language', the control should be announced as link as it takes the users to web page URL destination.

## User Impact:
Screen reader users will face difficulty in perceiving user interface component if incorrect role is assigned.

## Attachment:
![3_A11y_Visual Studio Code for MAC_Create New Project_Select a Language_Voicover](https://user-images.githubusercontent.com/98393966/167088441-03a179c9-769c-4258-b156-d3c9d5b72fdb.gif)

**Comment by MicroFish91** [comment]
Filed an issue that should cover this case.  You can track that issue here: https://github.com/microsoft/vscode/issues/149305

**Comment by fsteffi** [comment]
external bug logged:https://github.com/microsoft/vscode/issues/149305

as per external bug process this bug should be in open state until external bug is fixed

**Comment by MicroFish91** [comment]
Update: based on the linked issue's comment thread, this should be lowered to Severity 3 to match it's associated upstream issue severity

**Comment by fsteffi** [comment]
@MicroFish91  bugs can be lowered only after discussion with TCS SME

**Comment by MicroFish91** [comment]
For tracking purposes, comment by Isidor in [#149305](https://github.com/microsoft/vscode/issues/149305) indicates that the best issue to track against for this one is [#133169](https://github.com/microsoft/vscode/issues/133169).  There was mention that the severity of the issue was lowered and is being considered approved to drop by SME.

**Comment by MicroFish91** [comment]
Relevant comment [here](https://github.com/microsoft/vscode/issues/133169#issuecomment-1275639912) and [here](https://github.com/microsoft/vscode/issues/133169#issuecomment-1284812345) related to this issue.

**Comment by fsteffi** [comment]
approved to closed by SME "since the control is within another control, link control type is not needed"
ref bug https://github.com/microsoft/vscode/issues/133169

**Comment by bbarik06** [comment]
Closing the issue.
